client/extensions/zoninator/components/settings/zones-dashboard/zone-item.jsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { find, get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import sectionsModule from 'sections';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+const ZoneItem = ( { siteSlug, zone } ) => {
+	const sections = sectionsModule.get();
+	const section = find( sections, ( value => value.name === 'zoninator' ) );
+	const settingsPath = get( section, 'settings_path' );
```

Comment:
It's kind of annoying to duplicate this logic from `ZoneDashboard`. Furthermore, I don't think it should be invoked for every new render, when it doesn't even use any of the props. One option would be to make this a helper function, but I'm not even sure if it's worth it. Maybe just a constant?